src/lib.rs

```diff
@@ -47,15 +47,8 @@ pub struct NetworkingPlugin {
 
 impl Plugin for NetworkingPlugin {
     fn build(&self, app: &mut AppBuilder) {
-        let task_pool = app
-            .resources()
-            .get::<IoTaskPool>()
-            .expect("IoTaskPool resource not found")
-            .0
-            .clone();
-
-        app.add_resource(NetworkResource::new(
-            task_pool,
+        app.insert_resource(NetworkResource::new(
+            TaskPool::new(),
```

Comment:
Does that mean that Bevy does not support predefined task pools anymore?
I didn't notice this in the 0.5 release notes.

Reply:
It is not mentioned in migration guide, but new App Builder doesn't have Resources type anymore, so I think yes, it's not supported.

Reply:
`IoTaskPool` still exists.
https://github.com/bevyengine/bevy/blob/b17f8a4bce5551b418654fffb1fe97ff0f9852f0/crates/bevy_asset/src/lib.rs#L83

Reply:
I've found the Resources is now merged into `World`, so I brought back the task pool. Thank you.